org.mwc.cmap.legacy/build.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project name="cmap_legacy" default="build.jars" basedir=".">
+
+	<property name="javacSource" value="1.7"/>
+	<property name="javacTarget" value="1.7"/>
+	<property name="build.compiler" value="extJavac"/>
+	<condition property="dir_bootclasspath" value="${java.home}/../Classes">
+		<and>
+			<os family="mac"/>
+			<available file="${java.home}/../Classes" type="dir"/>
+		</and>
+	</condition>
+	<property name="dir_bootclasspath" value="${java.home}/lib"/>
+	<path id="path_bootclasspath">
+		<fileset dir="${dir_bootclasspath}">
+			<include name="*.jar"/>
+		</fileset>
+	</path>
+	<property name="bootclasspath" refid="path_bootclasspath"/>
+	<condition property="bundleBootClasspath" value="${JavaSE-1.7}"	>
+		<isset property="JavaSE-1.7"/>
+	</condition>
+	<property name="bundleBootClasspath" value="${bootclasspath}"/>
+
+	<target name="init">
+		<condition property="pluginTemp" value="${buildTempFolder}/plugins"		>
+			<isset property="buildTempFolder"/>
+		</condition>
+		<property name="pluginTemp" value="${basedir}"/>
+		<property name="temp.folder" value="${basedir}/temp.folder"/>
+	</target>
+
+	<target name="cmap_legacy.jar" depends="init" unless="cmap_legacy.jar" description="Create jar: org.mwc.cmap.legacy cmap_legacy.jar.">
+		<delete dir="${temp.folder}/cmap_legacy.jar.bin"/>
+		<mkdir dir="${temp.folder}/cmap_legacy.jar.bin"/>
+		<path id="cmap_legacy.jar.classpath">
+			<pathelement path="../org.mwc.cmap.jfreechart/libs/jcommon-1.0.23.jar"/>

```

Comment:
@ayeshare - is it possible to use a wildcard for these jars?  

If we're pulling in all the jars from the folder then using a wildcard will make the build less fragile if a .jar version changes.

Reply:
made this change